profiles/common/modules/custom/nexteuropa_varnish/src/PurgeRuleType.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * @file
+ * Definition of Drupal\nexteuropa_varnish\PurgeRuleType.
+ */
+
+namespace Drupal\nexteuropa_varnish;
+
+/**
+ * Enum of purge rule types.
+ */
+class PurgeRuleType {
+  const NODE = 'node';
+  const PATHS = 'paths';
+
+  /**
+   * Prevents the class from instantiated, it just acts as an enum.
+   */
+  private function __construct() {
```

Comment:
@tatarbj I think you are following your own personal opinion and habits, instead of allowing and learning a more modern style of code. I will not change this as I don't agree with your reasoning. There are other and far better approaches than what you are used to in Drupal 7. Time to get used to it with Drupal 8 around the corner.